saleor/settings.py

```diff
@@ -43,14 +43,9 @@
 EMAIL_URL = os.environ.get('EMAIL_URL', 'console://')
 email_config = dj_email_url.parse(EMAIL_URL)
 
-EMAIL_FILE_PATH = email_config['EMAIL_FILE_PATH']
-EMAIL_HOST_USER = email_config['EMAIL_HOST_USER']
-EMAIL_HOST_PASSWORD = email_config['EMAIL_HOST_PASSWORD']
-EMAIL_HOST = email_config['EMAIL_HOST']
-EMAIL_PORT = email_config['EMAIL_PORT']
-EMAIL_BACKEND = email_config['EMAIL_BACKEND']
-EMAIL_USE_TLS = email_config['EMAIL_USE_TLS']
+vars().update(email_config)
```

Comment:
Agreed to your point of searching for the variable.
The user may use either `EMAIL_USE_SSL` or `EMAIL_USE_TLS`.
What if the user wants `EMAIL_USE_SSL`? Default value of them are `False`.
`EMAIL_USE_TLS = False` doesn't make `EMAIL_USE_SSL=True`? Right.

So, do you think we have to add, `EMAIL_USE_SSL`?